NSelene/SeleneElement.cs

```diff
@@ -330,9 +329,26 @@ ReadOnlyCollection<IWebElement> SeleneContext.FindElements (By by)
             Should(Be.Visible);
             return this.ActualWebElement.FindElements(by);
         }
+        public void ExecuteScript(string script, params object[] args)
+        {
+            IJavaScriptExecutor js = (IJavaScriptExecutor)this.driver.Value;
+            js.ExecuteScript($@"
+                return (function(element, args) {{
+                    {script}
+                }})(arguments[0], arguments[1])", new object[] { this.ActualWebElement, args });
+        }
```

Comment:
do we really need it?
would not it be enough to have only the next "generic" one?

Reply:
seems like not enough...
only with generic we will have to do something like element.ExecuteScript<Object>(...) when we don't have a value... 

but do we really need then Generics here? why not return just an object like original webdriver method? which benefits do generics give us here?

Reply:
I really like the fact that the type casting is hidden inside the NSelene classes, instead that every usage has to do such. Maybe there are better solutions?

Now the users will also get type cast errors when they use the wrong type in the generic.

Reply:
Hm, If this makes the usage a lot easier, then seems reasonable... 

Can you provide a comparison of usage without generics and with generics? Several examples...

Reply:
There is no real difference, it more like
`long x = ele.ExecuteScript<long>("return 14;")`
vs
`long x = (long)ele.ExecuteScript("return 14;")`

Reply:
Yeah, I see now... Then let's keep the simpler "one method implementation" instead of "two methods – void and the generic one". 
It will be simpler, and probably more straightforward for the majority... Doing type case in a `(long) ...` style seems to be a standard that is known to everybody...